src/main_menu.cpp

```diff
@@ -243,6 +242,55 @@ std::vector<std::string> main_menu::load_file( const std::string &path,
     return result;
 }
 
+holiday main_menu:: get_holiday_from_time()
+{
+    std::tm *local_time;
+
+    std::time_t current_time = std::time( nullptr );
+    local_time = std::localtime( &current_time );
```

Comment:
Looks like this is needed to make lgtm.com happy.
```suggestion
    local_time = std::localtime_r( &current_time );
```

Reply:
I don't think `localtime_r` exists on Windows, so we probably have to use `localtime`.  I may be wrong, though.  Can't test it myself.